Makefile

```
@@ -1,4 +1,4 @@
-.PHONY: help lint format type-check security test pre-push clean test-tck test-tck-parallel
+.PHONY: help lint format type-check security test pre-push clean test-tck test-tck-parallel docstring-coverage test-network
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)